rspace/src/main/scala/coop/rchain/rspace/LMDBOps.scala

```diff
@@ -44,10 +44,6 @@ trait LMDBOps extends CloseOps {
       val ret: R = f(txn)
       txn.commit()
       ret
-    } catch {
```

Comment:
This deletion seems unrelated to the purpose of the PR.

Reply:
It was an addition to https://github.com/rchain/rchain/pull/1647 that I decided not to commit. I think it is no longer necessary and but it definitely should go in its own PR. Thanks for the catch.